src/index.tsx

```diff
@@ -11,14 +11,16 @@ const KEY_CODE = {
 
 type Props = {
   length?: number;
-  onChange: (data: string) => any;
+  onChange?: (data: string) => any;
+  onCompleted?: (data: string) => any;
   placeholder?: string;
   value?: string;
 };
 
 const ReactInputVerificationCode = ({
   length = 4,
   onChange,
+  onCompleted,
```

Comment:
```suggestion
  onCompleted = () => {},
```

This will avoid having to check `onCompleted` type everytime

Reply:
Alright, I did the same for onChange aswell.

Reply:
Alright, I did the same for onChange aswell